test/testlab.smoke.js

```diff
@@ -1,3 +1,9 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
 const expect = require('@loopback/testlab').expect;
```

Comment:
Let's remove this smoke test too, I don't see any value in such tests, plus they can confuse new LBN users and make them think that they should write similar tests in their apps.